.circleci/config.yml

```
@@ -2,17 +2,15 @@ version: 2
 jobs:
   build:
     docker:
-      - image: trussworks/circleci-docker-primary:9aeb08028f7d8413a463f4642526ab6a6e18832b
+      - image: trussworks/circleci-docker-primary:c542b22c7fb95db0a1bbe043928a457ae6fbeaca
         environment:
           - TEST_RESULTS: /tmp/test-results
     steps:
       - checkout
       - restore_cache:
           keys:
             - dot-cache-pre-commit-{{ checksum ".pre-commit-config.yaml" }}
-      - restore_cache:
-          keys:
-            - go-mod-sources-v2-{{ checksum "go.sum" }}
+            - go-mod-sources-v2-{{ checksum "go.sum" }}-{{ checksum "scripts/check-go-version" }}
```

This isn't the current path of the version script
```suggestion
            - go-mod-sources-v2-{{ checksum "go.sum" }}-{{ checksum "bin/check_go_version" }}
```